lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -453,6 +453,7 @@ sub job_next_previous_ajax ($self) {
                 DT_RowId => 'job_result_' . $job_id,
                 id => $job_id,
                 name => $job->name,
+                assigned_worker_id => $job->assigned_worker_id,
```

Comment:
```suggestion
                assigned_worker => $job->assigned_worker->name,
```

Reply:
But check how it impacts performance. Maybe it is a good idea to prefetch this column.

Reply:
How could we prefetch it?

Reply:
I guess that would need to be done in `lib/OpenQA/WebAPI/Controller/API/V1/Job.pm`